templates/account/fogg.tf.tmpl

```diff
@@ -47,6 +47,11 @@
   {{ template "datadog_provider" .Providers.Datadog }}
 {{ end }}
 
+{{ if .Providers.Tfe }}{{ if .Providers.Tfe.Enabled }}
```

Comment:
wonder if these conditionals should be inside the tfe template so theyre not repeated every fogg.tf template

Reply:
That makes sense, but given that the rest of the them follow this pattern, I would like to leave it as is for the sake of consistency and address it in a follow-up #462